packet/commands.py

```diff
@@ -15,22 +18,78 @@ def create_secret():
     print("Here's your random secure token:")
     print(token_hex())
 
+def get_member_username(member):
+    return member.uid
+
 @app.cli.command("create-packet")
-def create_packet():
+@click.argument("freshmen_csv")
+def create_packet(freshmen_csv):
     """
-    Example/test code for adding a new packet to the database.
+    Creates all database entries needed to start a new packet season.
     """
-    print("Generating new rows...")
+    print("Welcome to the packet season creation CLI", end="\n\n")
 
-    freshman = Freshman(rit_username="bob1234", name="Bob Freshy", onfloor=True)
-    db.session.add(freshman)
+    rawDate = None
+    while rawDate is None:
+        try:
+            dateStr = input("Input the first day of packet season (format: MM/DD/YYYY): ")
+            rawDate = datetime.strptime(dateStr, "%m/%d/%Y").date()
+        except ValueError:
+            pass
 
-    packet = Packet(freshman=freshman)
-    db.session.add(packet)
+    start = datetime.combine(rawDate, time(hour=19))
+    end = datetime.combine(rawDate, time(hour=23, minute=59)) + timedelta(days=14)
+    print("start = {}; end = {}".format(start, end), end="\n\n")
 
-    db.session.add(UpperSignature(packet=packet, member="somehuman"))
-    db.session.add(UpperSignature(packet=packet, member="reeehuman", eboard=True))
+    eboard = list(map(get_member_username, ldap_get_eboard()))
+    onfloor = list(map(get_member_username, ldap_get_live_onfloor()))
+    print("eboard members = {}".format(eboard))
+    print("onfloor members = {}".format(onfloor), end="\n\n")
 
-    db.session.commit()
+    print("Parsing freshmen csv...")
+    allFreshmen = {}
+    onfloorFreshmen = {}
+
+    try:
+        with open(freshmen_csv) as freshmenFile:
+            for row in freshmenFile:
+                row = row.strip().split(",")
+
+                allFreshmen[row[3]] = row[0]
+
+                if row[1] == "TRUE":
+                    onfloorFreshmen[row[3]] = row[0]
+
+    except Exception as e:
+        print("Things broke")
+        raise e
+
+    print("All freshmen = {}".format(allFreshmen))
+    print("On floor freshmen = {}".format(onfloorFreshmen), end="\n\n")
 
+    if input("Data ready; create the season? (Y/n): ").lower() != "y":
+        return
+
+    # Go ahead and create all Freshman entries now so FreshSignatures can be created iteratively
+    freshmen = {}
```

Comment:
I might name this something more specific like `freshman_db_entries` to distinguish it from `allFreshmen`.